scripts/generate_import_mr_sessions_batch_command.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
```

Comment:
Delete this file - it's in `scripts/` and it also doesn't appear to be changed from the template.